src/benchmarks/gc/README.md

```diff
@@ -20,7 +20,9 @@ The general workflow when using the GC infra is:
 ### Install python 3.7+
 
 On Windows, just go to https://www.python.org/downloads/ and run the installer.
-On other systems it’s better to use your system’s package manager.
+It's recommended to install a 64-bit version if possible, but not required.
```

Comment:
This is already in the upstream. I think you need to rebase to pick documentation changes.